templates/zerver/help/include/sidebar.md

```diff
@@ -107,45 +112,45 @@
 <!-- Zulip in a terminal -->
 <!-- Connect to Zulip over IRC/etc (not implemented?) -->
 
-## Organization Settings
-
-* [Change your organization settings](/help/change-your-organization-settings)
-* [Change your organization's name](/help/change-your-organizations-name)
-* [Change your organization's description](/help/change-your-organizations-description)
-* [Change your organization's avatar](/help/change-your-organizations-avatar)
-* [Restrict user email addresses to certain domains](/help/restrict-user-email-addresses-to-certain-domains)
-* [Allow anyone to join without an invitation](/help/allow-anyone-to-join-without-an-invitation)
-* [Only allow admins to invite new users](/help/only-allow-admins-to-invite-new-users)
-* [Only allow admins to create new streams](/help/only-allow-admins-to-create-new-streams-feature)
-* [Only allow admins to add emoji](/help/only-allow-admins-to-add-emoji)
-* [Allow image/link previews](/help/allow-image-link-previews)
+# Org administration
```

Comment:
Administration

Reply:
I think Org administration scans better with "Using Zulip"? I could also see "Zulip administration", if we don't like the abbreviation.

Reply:
Yeah, "Zulip administration" would be better.  I think it's a bad idea to abbreviate in only some places.